sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilderSamples.cs

```diff
@@ -18,7 +17,7 @@ public static class QueryBuilderSamples
         public static void main()
         {
             //"SELECT * FROM DIGITALTWINS";
-            AdtQuery query1 = new AdtQuery().Select("*").From(AdtCollection.DigitalTwins);
+            AdtQueryBuilder query1 = new AdtQueryBuilder().Select("*").From(AdtCollection.DigitalTwins).Build();
 
             ////"SELECT * FROM DIGITALTWINS WHERE Temperature <= 50";
```

Comment:
You can start cleaning up the samples (removing commented lines) and keep adding to them as you add more support for different scenarios.